<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BiLap103-PC" failures="1" tests="11" name="TestCases.CheckPromotion" time="731.448" errors="0" timestamp="2021-07-07T15:01:08 EET" skipped="0">
  <testcase classname="TestCases.CheckPromotion" name="checkQuantityToFreeGoods" time="70.080"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkQuantityToValue" time="68.646"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkQuantityToFreeGoods" time="55.776"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkQuantityToPercantage" time="68.499"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkValueToPercantage" time="66.295"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkValueToValue" time="70.717"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkValueToNewPrice" time="70.130"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkQuantityToFreeGoods" time="69.778"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkValueToFreeGoods" time="66.949"/>
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkQuantityToValue" time="69.439">
    <failure message="Something wrong with value promotion expected [20.0] but found [0.0]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Something wrong with value promotion expected [20.0] but found [0.0]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEquals(Assert.java:701)
at TestCases.CheckPromotion.checkQuantityToValue(CheckPromotion.java:158)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkQuantityToValue -->
  <system-out/>
  <testcase classname="TestCases.CheckPromotion" name="checkQuantityToNewPrice" time="55.139"/>
  <system-out/>
</testsuite> <!-- TestCases.CheckPromotion -->
